### PR TITLE
[NUITizenGallery] Add ShadowFrameTest1 sample

### PR DESCRIPTION
**Pull request by hinohie**
Sample for use shadow / borderline / cornerradius

Screenshot from 2021-09-27 17-49-10

Signed-off-by: Eunki, Hong <eunkiki.hong@samsung.com

### CODE REVIEW

File: NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml [hunk 1]
```diff
@@ -0,0 +1,672 @@
...
+                    x:Name="CornerRadiusView"
+                    WidthSpecification="-1"
+                    HeightSpecification="-2"
+                >
+                    <View.Layout>
+                        <LinearLayout LinearOrientation="Vertical" LinearAlignment="Begin" CellPadding="5,5"/>
+                    </View.Layout>
+                    <b:TextLabel
+                        x:Name="CornerRadiusViewHeader"
+                        WidthSpecification="-1"
```
Comment (everLEEst) [inline NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml:66]: use WidthSpecification="{Static LayoutParamPolicies.MatchParent}"

File: NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml [hunk 2]
```diff
@@ -0,0 +1,672 @@
...
+                        </b:View>
+                     </b:View>
+                </b:View>
+
+
+                <!-- Shadow Offset -->
+                <b:View
+                    x:Name="ShadowOffsetView"
+                    WidthSpecification="-1"
+                    HeightSpecification="-2"
```
Comment (everLEEst) [inline NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml:167]: use WidthSpecification="{Static LayoutParamPolicies.MatchParent}" HeightSpecification="{Static LayoutParamPolicies.WrapContent}"

File: NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml [hunk 3]
```diff
@@ -0,0 +1,672 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<b:View x:Class="NUITizenGallery.ShadowFrameTest1Page"
+  xmlns="http://tizen.org/Tizen.NUI/2018/XAML"
+  xmlns:b="clr-namespace:Tizen.NUI.BaseComponents;assembly=Tizen.NUI"
+  xmlns:nui="clr-namespace:Tizen.NUI.Components;assembly=Tizen.NUI.Components"
```
Comment (everLEEst) [inline NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml:5]: hope we use same namespace,
b->base
nui->comp

File: NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml [hunk 4]
```diff
@@ -0,0 +1,672 @@
...
+                        HorizontalAlignment="Begin"
+                        VerticalAlignment="Center"
+                        PointSize="25"
+                        TextColor="#fe7c57"
+                        Text="ShadowOffset"
+                    />
+                    <b:View
+                        x:Name="ShadowOffsetViewContent"
+                        WidthSpecification="-1"
+                        HeightSpecification="-2"
```
Comment (everLEEst) [inline NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml:184]: use WidthSpecification="{Static LayoutParamPolicies.MatchParent}" HeightSpecification="{Static LayoutParamPolicies.WrapContent}"

File: NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml [hunk 5]
```diff
@@ -0,0 +1,672 @@
...
+                    x:Name="ShadowOffsetView"
+                    WidthSpecification="-1"
+                    HeightSpecification="-2"
+                >
+                    <View.Layout>
+                        <LinearLayout LinearOrientation="Vertical" LinearAlignment="Begin" CellPadding="5,5"/>
+                    </View.Layout>
+                    <b:TextLabel
+                        x:Name="ShadowOffsetViewHeader"
+                        WidthSpecification="-1"
```
Comment (everLEEst) [inline NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml:174]: use WidthSpecification="{Static LayoutParamPolicies.MatchParent}"

File: NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml [hunk 6]
```diff
@@ -0,0 +1,672 @@
...
+                        HeightSpecification="-2"
+                    >
+                        <View.Layout>
+                            <LinearLayout LinearOrientation="Vertical" LinearAlignment="Begin" CellPadding="5,5"/>
+                        </View.Layout>
+                        <b:View
+                            x:Name="ShadowOffsetView1"
+                            WidthSpecification="-1"
+                            HeightSpecification="-2"
```
Comment (everLEEst) [inline NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml:192]: use WidthSpecification="{Static LayoutParamPolicies.MatchParent}" HeightSpecification="{Static LayoutParamPolicies.WrapContent}"

File: NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml [hunk 7]
```diff
@@ -0,0 +1,672 @@
...
+                                HorizontalAlignment="Begin"
+                                VerticalAlignment="Center"
+                                Text="Off X : "
+                            />
+                            <nui:Slider x:Name="ShadowOffsetX"
+                                        MinValue="-40"
+                                        MaxValue="40"
+                                        CurrentValue="-20"
+                                        ThumbSize="8,8"
+                                        WidthSpecification="-1"
```
Comment (everLEEst) [inline NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml:210]: use WidthSpecification="{Static LayoutParamPolicies.MatchParent}"

File: NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml [hunk 8]
```diff
@@ -0,0 +1,672 @@
...
+                                HorizontalAlignment="Begin"
+                                VerticalAlignment="Center"
+                                Text="Off Y : "
+                            />
+                            <nui:Slider x:Name="ShadowOffsetY"
+                                        MinValue="-30"
+                                        MaxValue="30"
+                                        CurrentValue="20"
+                                        ThumbSize="8,8"
+                                        WidthSpecification="-1"
```
Comment (everLEEst) [inline NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml:225]: use WidthSpecification="{Static LayoutParamPolicies.MatchParent}"

File: NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml [hunk 9]
```diff
@@ -0,0 +1,672 @@
...
+                            />
+                        </b:View>
+                     </b:View>
+                </b:View>
+
+                <!-- Shadow Toggle -->
+                <b:View
+                    x:Name="ShadowToggleView"
+                    WidthSpecification="-1"
+                    HeightSpecification="-2"
```
Comment (everLEEst) [inline NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml:235]: use WidthSpecification="{Static LayoutParamPolicies.MatchParent}" HeightSpecification="{Static LayoutParamPolicies.WrapContent}"

File: NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml [hunk 10]
```diff
@@ -0,0 +1,672 @@
...
+                >
+                    <View.Layout>
+                        <LinearLayout LinearOrientation="Vertical" LinearAlignment="Begin" CellPadding="5,5"/>
+                    </View.Layout>
+                    <b:TextLabel
+                        x:Name="ShadowToggleViewHeader"
+                        WidthSpecification="-1"
```
Comment (everLEEst) [inline NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml:242]: use WidthSpecification="{Static LayoutParamPolicies.MatchParent}"

File: NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml [hunk 11]
```diff
@@ -0,0 +1,672 @@
...
+                        HorizontalAlignment="Begin"
+                        VerticalAlignment="Center"
+                        PointSize="25"
+                        TextColor="#fe7c57"
+                        Text="ShadowOpacity/BlurRadius"
+                    />
+                    <b:View
+                        x:Name="ShadowToggleViewContent"
+                        WidthSpecification="-1"
+                        HeightSpecification="-2"
```
Comment (everLEEst) [inline NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml:252]: use WidthSpecification="{Static LayoutParamPolicies.MatchParent}" HeightSpecification="{Static LayoutParamPolicies.WrapContent}"

File: NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml [hunk 12]
```diff
@@ -0,0 +1,672 @@
...
+                    >
+                        <View.Layout>
+                            <LinearLayout LinearOrientation="Vertical" LinearAlignment="Begin" CellPadding="5,5"/>
+                        </View.Layout>
+                        <b:View
+                            x:Name="ShadowToggleView1"
+                            WidthSpecification="-1"
+                            HeightSpecification="-2"
```
Comment (everLEEst) [inline NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml:260]: use WidthSpecification="{Static LayoutParamPolicies.MatchParent}" HeightSpecification="{Static LayoutParamPolicies.WrapContent}"

File: NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml [hunk 13]
```diff
@@ -0,0 +1,672 @@
...
+                                HorizontalAlignment="Begin"
+                                VerticalAlignment="Center"
+                                Text="Opacity"
+                            />
+                            <nui:Slider x:Name="ShadowOpacity"
+                                        MinValue="0"
+                                        MaxValue="255"
+                                        CurrentValue="48"
+                                        ThumbSize="8,8"
+                                        WidthSpecification="-1"
```
Comment (everLEEst) [inline NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml:278]: use WidthSpecification="{Static LayoutParamPolicies.MatchParent}"

File: NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml [hunk 14]
```diff
@@ -0,0 +1,672 @@
...
+                                HorizontalAlignment="Begin"
+                                VerticalAlignment="Center"
+                                Text="Blur R"
+                            />
+                            <nui:Slider x:Name="ShadowBlurRadius"
+                                        MinValue="0"
+                                        MaxValue="50"
+                                        CurrentValue="10"
+                                        ThumbSize="8,8"
+                                        WidthSpecification="-1"
```
Comment (everLEEst) [inline NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml:293]: use WidthSpecification="{Static LayoutParamPolicies.MatchParent}"

File: NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml [hunk 15]
```diff
@@ -0,0 +1,672 @@
...
+                            />
+                        </b:View>
+
+                        <b:View
+                            x:Name="ShadowToggleView2"
+                            WidthSpecification="-1"
+                            HeightSpecification="-2"
```
Comment (everLEEst) [inline NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml:300]: use WidthSpecification="{Static LayoutParamPolicies.MatchParent}" HeightSpecification="{Static LayoutParamPolicies.WrapContent}"

File: NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml [hunk 16]
```diff
@@ -0,0 +1,672 @@
...
+                            />
+                        </b:View>
+                     </b:View>
+                </b:View>
+
+                <!-- Shadow Color -->
+                <b:View
+                    x:Name="ShadowColorView"
+                    WidthSpecification="-1"
+                    HeightSpecification="-2"
```
Comment (everLEEst) [inline NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml:323]: use WidthSpecification="{Static LayoutParamPolicies.MatchParent}" HeightSpecification="{Static LayoutParamPolicies.WrapContent}"

File: NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml [hunk 17]
```diff
@@ -0,0 +1,672 @@
...
+                >
+                    <View.Layout>
+                        <LinearLayout LinearOrientation="Vertical" LinearAlignment="Begin" CellPadding="5,5"/>
+                    </View.Layout>
+                    <b:TextLabel
+                        x:Name="ShadowColorViewHeader"
+                        WidthSpecification="-1"
```
Comment (everLEEst) [inline NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml:330]: use WidthSpecification="{Static LayoutParamPolicies.MatchParent}"

File: NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml [hunk 18]
```diff
@@ -0,0 +1,672 @@
...
+                        HorizontalAlignment="Begin"
+                        VerticalAlignment="Center"
+                        PointSize="25"
+                        TextColor="#fe7c57"
+                        Text="ShadowColor"
+                    />
+                    <b:View
+                        x:Name="ShadowColorViewContent"
+                        WidthSpecification="-1"
+                        HeightSpecification="-2"
```
Comment (everLEEst) [inline NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml:340]: use WidthSpecification="{Static LayoutParamPolicies.MatchParent}" HeightSpecification="{Static LayoutParamPolicies.WrapContent}"

File: NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml [hunk 19]
```diff
@@ -0,0 +1,672 @@
...
+                    >
+                        <View.Layout>
+                            <LinearLayout LinearOrientation="Vertical" LinearAlignment="Begin" CellPadding="5,5"/>
+                        </View.Layout>
+                        <b:View
+                            x:Name="ShadowColorView1"
+                            WidthSpecification="-1"
+                            HeightSpecification="-2"
```
Comment (everLEEst) [inline NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml:348]: use WidthSpecification="{Static LayoutParamPolicies.MatchParent}" HeightSpecification="{Static LayoutParamPolicies.WrapContent}"

.... all negative numeric need to be changed this. thank you.

**Comment by hinohie** [comment]
I'll be back.

**Comment by hinohie** [comment]
I'm back.

File: NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml (hunk 3 above)
Comment (hinohie) [reply NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml]: I check up that we don't need to setup base:View or comp:Slider.

File: NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml [hunk 20]
```diff
@@ -0,0 +1,672 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<b:View x:Class="NUITizenGallery.ShadowFrameTest1Page"
```
[the remark this answers is not in the document]
Comment (hinohie) [reply NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml]: Done

**Review by JSUYA — APPROVED** [review]
There doesn't seem to be a problem when I review it.

**Pull request merged**